lua/telescope/_extensions/tmux/sessions.lua

```diff
@@ -57,17 +57,24 @@ local sessions = function(opts)
         finder = finders.new_table {
             results = user_formatted_session_names,
             entry_maker = function(result)
```

Comment:
Hi, thanks for the PR. I think it would be better to allow passing the entire `entry_maker` via `opts`. It would support a lot more use cases that way